front/pages/api/stripe/webhook.ts

```diff
@@ -411,5 +435,104 @@ async function revokeUsersForDowngrade(workspaceId: number) {
   for (const membership of [...adminMemberships, ...nonAdminMemberships]) {
     await membership.update({ role: "revoked" });
   }
+
+  // revoke invitations
+  const workspaceId = auth.workspace()?.id;
+  if (!workspaceId) {
+    throw new Error("Cannot get workspace.");
+  }
+  await MembershipInvitation.update(
+    { status: "revoked" },
+    { where: { workspaceId } }
+  );
+}
+
+async function archiveConnectedAgents(auth: Authenticator) {
+  const agentConfigurations = await getAgentConfigurations(auth, "super_user");
```

Comment:
Maybe we just need to rename this `admin` and let the admin go through that @philipperolet

Reply:
`admin_internal` to be super clear